languages/CallNumberFirst/en-gb.ini

```diff
@@ -1 +1 @@
-@parent_ini = "CallNumberFirst/en.ini"
+
```

Comment:
Do these empty files even need to exist if the fallback is now handled by configuration? Do you think there's a value in explicitly creating an empty file, or should we allow the code to fall back from an empty file without complaining (assuming that's not what it does already)?

Reply:
> Do these empty files even need to exist if the fallback is now handled by configuration?

Language files do not need to exist for non-default text domains. So these files could be removed.

For the default text domain at least one of the registered language directories needs to contain a language file for a requested locale.

Reply:
That's great -- and definitely another argument in favor of this approach, since it's a nuisance to have to create lots of empty language files with `@parent_ini` directives every time a new text domain is created. I'd definitely favor deleting the empty files from the PR.